janitor/src/main/scala/com/broadinstitute/dsp/janitor/DbReader.scala

```diff
@@ -60,6 +60,7 @@ object DbReader {
       FROM KUBERNETES_CLUSTER kc
       WHERE
         kc.status != "DELETED" AND
+        kc.cloudProvider = "GCP" AND
```

Comment:
I would add a comment above explaining that we only apply janitor on google resources, and so this filter makes sense for this cron-job only